src/main/java/org/sagebionetworks/web/client/widget/entity/tabs/ChallengeTab.java

```diff
@@ -119,15 +119,9 @@ public void showCreateNewEvaluationEditor(String entityId){
 
 	public void updateActionMenuCommands(){
 		actionMenuWidget.setActionVisible(Action.ADD_EVALUATION_QUEUE, permissions.getCanEdit());
-		if(DisplayUtils.isInTestWebsite(cookies)){
-			actionMenuWidget.setActionListener(Action.ADD_EVALUATION_QUEUE, (Action action) -> {
-				showCreateNewEvaluationEditor(entityId);
-			});
-		}else {
-			actionMenuWidget.setActionListener(Action.ADD_EVALUATION_QUEUE, (Action action) -> {
-				showOldAddEvaluationQueueModal();
-			});
-		}
+		actionMenuWidget.setActionListener(Action.ADD_EVALUATION_QUEUE, (Action action) -> {
+			showCreateNewEvaluationEditor(entityId);
+		});
 	}
 
 	private void showOldAddEvaluationQueueModal() {
```

Comment:
Is showOldAddEvaluationQueueModal() dead code after this change?

Reply:
No, it is still being used to be backwards compatible with Evaluations that have the old SubmissionQuotas.
I did a code review with @marcomarasca , and he suggested a better plan to allow users to opt-in to the conversion via a REST API call instead of a mass migration. 
On the UI side, I will need to add a button on the old evaluation editor to call that API, but given that it'll be thrown away eventually, it probably doesn't need to look pretty.